spec/lib/reporting/identity_verification_report_spec.rb

```diff
@@ -67,12 +84,15 @@
   describe '#data' do
     it 'counts unique users per event as a hash' do
       expect(report.data).to eq(
```

Comment:
in hindsight, maybe we could have had a separate hash for the derived numbers (results module) but in the end same same